operator/apis/platform/v1alpha1/central_types.go

```diff
@@ -194,15 +194,12 @@ type CentralDBSpec struct {
 }
 
 // CentralDBEnabled is a type for values of spec.central.db.enabled.
-//+kubebuilder:validation:Enum=Default;Enabled
+//+kubebuilder:validation:Enum=Default
 type CentralDBEnabled string
 
 const (
-	// CentralDBEnabledDefault configures the central to use rocksdb
-	// TODO(ROX-13557): in 3.74.0 this should say that Default enables PostgreSQL.
+	// CentralDBEnabledDefault configures the central to use PostgreSQL database (Technology Preview)
 	CentralDBEnabledDefault CentralDBEnabled = "Default"
-	// CentralDBEnabledTrue configures the central to use a PostgreSQL database (Technology Preview)
-	CentralDBEnabledTrue CentralDBEnabled = "Enabled"
```

Comment:
I also don't think it's a good idea to remove the "Enabled" value. All our existing ACS instances in ACSCS use "Enabled" atm.

I think we should keep both values for now, the only difference being that "Default" now also means "Enabled". After the 3.74 release, we can omit setting this field in `fleetshard-sync`. Then in the 3.75 release we can remove "isEnabled" altogether.